cocos/platform/mac/CCApplication-mac.mm

```diff
@@ -83,7 +83,6 @@ static long getCurrentMillSecond()
         lastTime = getCurrentMillSecond();
         
         director->mainLoop();
-        glview->pollEvents();
```

Comment:
Should check other platforms too.

Reply:
It's available only on mac

Reply:
I mean should check the implemetation of windows and linux.